src/stan/model/indexing/lvalue.hpp

```diff
@@ -14,9 +14,33 @@ namespace stan {
 namespace model {
 
 /**
- * Assign the specified rvalue to the specified lvalue.  The index
- * list's type must be `nil_index_list`, but its value will be
- * ignored.  The last two arguments are also ignored.
+ * Indexing Notes:
+ * The different index types:
+ * index_uni - A single cell.
+ * index_multi - Random access index.
```

Comment:
Yeah I like your name more, I was thinking random access just because the index can come in like `{1,1,4,8,8,3,9}` etc. so it's like a random access iterator